src/DonorPerfect.php

```diff
@@ -218,8 +218,9 @@ public function call(string $action, array $parameters)
                 $value = "N'" . implode($value, '|') . "'";
             } else {
                 // Ensure quotes are doubled for escaping purposes
+                // + in param will be interpreted as a space and will generate an error
                 // @see https://api.warrenbti.com/2020/08/03/apostrophes-in-peoples-names/
-                $value = str_replace(["'", '"', '%'], ["''", '', '%25'], $value);
+                $value = str_replace(["'", '"', '%', '+'], ["''", '', '%25', ''], $value);
```

Comment:
```suggestion
                $value = str_replace(["'", '"', '%', '+'], ["''", '', '%25', '%2b'], $value);
```
We should probably just URL encode the `+` rather than remove it outright

Reply:
@RonAbarbanel any thoughts?